packages/nodes-base/nodes/Set/test/v2/raw.test.ts

```diff
@@ -0,0 +1,145 @@
+import { get } from 'lodash';
+import { constructExecutionMetaData } from 'n8n-core';
+import {
+	NodeOperationError,
+	type IDataObject,
+	type IExecuteFunctions,
+	type IGetNodeParameterOptions,
+	type INode,
+} from 'n8n-workflow';
+
+import { type SetNodeOptions } from '../../v2/helpers/interfaces';
+import * as utils from '../../v2/helpers/utils';
+import { execute } from '../../v2/raw.mode';
+
+export const node: INode = {
+	id: '11',
+	name: 'Set Node',
+	type: 'n8n-nodes-base.set',
+	typeVersion: 3,
+	position: [42, 42],
+	parameters: {
+		mode: 'manual',
+		fields: {
+			values: [],
+		},
+		include: 'none',
+		options: {},
+	},
+};
+
+export const createMockExecuteFunction = (
```

Comment:
Since we don't use this anywhere else let's remove export
```suggestion
const createMockExecuteFunction = (
```